IGraphics/IGraphics.cpp

```diff
@@ -986,6 +986,7 @@ void IGraphics::OnMouseDown(const std::vector<IMouseInfo>& points)
   {
     mMouseDownX = points[0].x;
     mMouseDownY = points[0].y;
+    mMouseIsDown = true;
```

Comment:
Why not just put this at the top of this function? Why does it matter is there is a single point or multiple?

Reply:
Also mMouseDown should be made private with a const accessor function.